### ISSUE TITLE
Improve Log Quality

### ISSUE THREAD

**Issue by davidsmejia** [AlexsLemonade/refinebio#1670]
### Context

The logs are currently overwhelming and hard to parse. 

### Problem or idea

Processor jobs are failing and its difficult to find the related information.

### Solution or next step

Consider removing anything from a logger function that can be found via query etc, standardize how logs appear for easier grepping / utilization.

**Comment by cgreene** [comment]
I would also suggest reporting only errors to sentry and having anything with the error log level be at the "OMG EMAIL ME" level of concern. We can set up sentry to email us or notify a slack channel, so we can keep better visibility on the very serious issues.

**Comment by kurtwheeler** [comment]
Specifically I would appreciate it if we ripped out anywhere that we log a collection. These are long and make the log lines really really long. Additionally any log that is coming out of a job should have a job_id field logged and preferably that field would always be first.

**Comment by cgreene** [comment]
I'm also not sure I'd ever _remove_ a logging call unless it's absolutely absurd. Instead it would be better to use info/debug loglevels so we can turn them on for cases where we want to debug.

**Comment by kurtwheeler** [comment]
I didn't mean remove logging calls, I meant remove the part where we log long lists of ids. Right now our logs contain lines that are several screens worth of text. It's nearly impossible to get any useful information out of it or even just scroll past it using `less`.

**Comment by cgreene** [comment]
@kurtwheeler Gotcha! If they are real errors maybe good to pass it as context to sentry, but I agree with not writing it to the message. FWIW it also breaks sentry's message grouping if messages come in with different text for the same error.

**Comment by arielsvn** [comment]
Related https://github.com/AlexsLemonade/refinebio/issues/1396: it'd be nice to also make it easier/faster to search the logs.
